.github/workflows/build_test.yml

```diff
@@ -408,3 +408,30 @@ jobs:
     # ICL implies SKX, CLX and CNL
     - name: Run SIMD tests (Ice Lake)
       run: sde -icl -- python runtests.py -n -v -- -k test_simd
+
+  intel_spr_sde_test:
+    needs: [smoke_test]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435 # v4.5.0
+      with:
+        python-version: ${{ env.PYTHON_VERSION }}
+    - name: Install Intel SDE
+      run: |
+        curl -o /tmp/sde.tar.xz https://downloadmirror.intel.com/732268/sde-external-9.7.0-2022-05-09-lin.tar.xz
+        mkdir /tmp/sde && tar -xvf /tmp/sde.tar.xz -C /tmp/sde/
+        sudo mv /tmp/sde/* /opt/sde && sudo ln -s /opt/sde/sde64 /usr/bin/sde
+    - name: Install dependencies
+      run: |
+        python -m pip install -r test_requirements.txt
+        sudo apt install gcc-12 g++-12
+    - name: Build
+      run: |
+        export CC=/usr/bin/gcc-12
+        export CXX=/usr/bin/g++-12
+        python setup.py install
+        sde -spr -- python runtests.py -n -v
```

Comment:
Could you split this into two steps?

I think you may need to only run the ufunc and linalg tests, with a comment that this is very slow

Reply:
Done.